<?xml version="1.0"?>

<launch>
  <arg name="load_param" default="true" />
  <arg name="tests_type" default="simulation" />
  <arg name="task_name" default="can_pick_and_place" />
  <arg name="tree_name" default="simulation_can_pick_and_place_learning" />
 
  <param name="task_name" type="string" value="$(arg task_name)" />
  <param name="tree_name" type="string" value="$(arg tree_name)" />
  <param name="tests_type" type="string" value="$(arg tests_type)" />

  <include if = "$(arg load_param)" file="$(find battery_cell_tests)/$(arg task_name)/launch/load_test_params.launch">
    <arg name="tests_type" default="$(arg tests_type)" />
  </include>

  <node name="run_opt_tests_node" pkg="battery_cell_tests" type="run_opt_tests.py" output="screen"/>
  <node name="rosparam_to_rostopic" pkg="kr_50_r2500_tests" type="rosparam_to_rostopic.py" output="screen"/>

</launch>
